source/MRViewer/MRRibbonFontManager.cpp

```diff
@@ -30,16 +30,22 @@ void RibbonFontManager::loadAllFonts( ImWchar* charRanges, float scaling )
 
     const ImWchar iconRanges[] = { 0xe005, 0xf8ff, 0 };
 
-    for ( int i = 0; i<int( FontType::Count ); ++i )
+    std::vector<CustomGlyph> customGlyphs;
+
+    for ( int i = 0; i< int( FontType::Count ); ++i )
     {
         if ( i == int( FontType::Monospace ) )
             loadFont_( FontType::Monospace, ImGui::GetIO().Fonts->GetGlyphRangesDefault(), scaling );
         if ( i == int( FontType::Icons ) )
             loadFont_( FontType::Icons, iconRanges, scaling );
         else
             loadFont_( FontType( i ), charRanges, scaling );
+
+        addCustomGlyphs_( FontType( i ), scaling, customGlyphs );
```

Comment:
to all fonts?

Reply:
Excluded icons font.

Reply:
mb move font selection logic to this loop, out of the function?